src/shared/cpu-set-util.c

```diff
@@ -290,3 +337,22 @@ int cpu_set_from_dbus(const uint8_t *bits, size_t size, CPUSet *set) {
         *set = TAKE_STRUCT(s);
         return 0;
 }
+
+int cpu_mask_add_all(CPUSet *mask) {
+        long m;
+
+        assert(mask);
+
+        m = sysconf(_SC_NPROCESSORS_ONLN);
```

Comment:
this can actually fail (since it goes to sysfs internally, which might be inaccessible, who knows. please handle error and propagate it